test/tools/ossfuzz/solProtoFuzzer.cpp

```diff
@@ -37,160 +38,8 @@ using namespace std;
-evmc::result deployContract(EVMHost& _hostContext, bytes const& _code)
-{
-	evmc_message message = initializeMessage(_code);
-	message.kind = EVMC_CREATE;
-	return _hostContext.call(message);
-}
-
-std::pair<bytes, Json::Value> compileContract(
-	std::string _sourceCode,
-	std::string _contractName,
-	std::map<std::string, solidity::util::h160> const& _libraryAddresses = {},
-	frontend::OptimiserSettings _optimization = frontend::OptimiserSettings::minimal()
-)
-{
-	try
-	{
-		// Compile contract generated by the proto fuzzer
-		SolidityCompilationFramework solCompilationFramework;
-		return std::make_pair(
-			solCompilationFramework.compileContract(_sourceCode, _contractName, _libraryAddresses, _optimization),
-			solCompilationFramework.getMethodIdentifiers()
-		);
-	}
-	// Ignore stack too deep errors during compilation
-	catch (evmasm::StackTooDeepException const&)
-	{
-		return std::make_pair(bytes{}, Json::Value(0));
-	}
-}
-
-evmc::result deployAndExecute(EVMHost& _hostContext, bytes _byteCode, std::string _hexEncodedInput)
-{
-	// Deploy contract and signal failure if deploy failed
-	evmc::result createResult = deployContract(_hostContext, _byteCode);
-	solAssert(
-		createResult.status_code == EVMC_SUCCESS,
-		"Proto solc fuzzer: Contract creation failed"
-	);
-
-	// Execute test function and signal failure if EVM reverted or
-	// did not return expected output on successful execution.
-	evmc::result callResult = executeContract(
-		_hostContext,
-		fromHex(_hexEncodedInput),
-		createResult.create_address
-	);
-
-	// We don't care about EVM One failures other than EVMC_REVERT
-	solAssert(callResult.status_code != EVMC_REVERT, "Proto solc fuzzer: EVM One reverted");
-	return callResult;
-}
-
-evmc::result compileDeployAndExecute(
-	std::string _sourceCode,
-	std::string _contractName,
-	std::string _methodName,
-	frontend::OptimiserSettings _optimization,
-	std::string _libraryName = {}
-)
-{
-	bytes libraryBytecode;
-	Json::Value libIds;
-	// We target the default EVM which is the latest
-	langutil::EVMVersion version = {};
-	EVMHost hostContext(version, evmone);
-	std::map<std::string, solidity::util::h160> _libraryAddressMap;
-
-	// First deploy library
-	if (!_libraryName.empty())
-	{
-		tie(libraryBytecode, libIds) = compileContract(
-			_sourceCode,
-			_libraryName,
-			{},
-			_optimization
-		);
-		// Deploy contract and signal failure if deploy failed
-		evmc::result createResult = deployContract(hostContext, libraryBytecode);
-		solAssert(
-			createResult.status_code == EVMC_SUCCESS,
-			"Proto solc fuzzer: Library deployment failed"
-		);
-		_libraryAddressMap[_libraryName] = EVMHost::convertFromEVMC(createResult.create_address);
-	}
-
-	auto [bytecode, ids] = compileContract(
-		_sourceCode,
-		_contractName,
-		_libraryAddressMap,
-		_optimization
-	);
-
-	return deployAndExecute(
-		hostContext,
-		bytecode,
-		ids[_methodName].asString()
-	);
-}
+static vector<uint8_t> const expectedOutput(32, 0);
```

Comment:
If you want those as globals, then maybe it should rather be
```
static array<uint8_t, 32> const expectedOutput{{}};
```

Reply:
I will incorporate these changes. Hence dismissing your review so this PR does not get merged because it is approved :)

Reply:
This is now removed in all instances since we always have the same expectation (zero word).